base/uk.ac.stfc.isis.ibex.synoptic/src/uk/ac/stfc/isis/ibex/synoptic/SynopticWriter.java

```diff
@@ -38,18 +39,21 @@
 public class SynopticWriter extends TransformingWriter<SynopticDescription, String> {
 	
 	private SettableUpdatedValue<Boolean> canSave = new SettableUpdatedValue<>();
+    private final Observable<String> schema;
 
-	public SynopticWriter(Writable<String> destination) {
+    public SynopticWriter(Writable<String> destination, Observable<String> schema) {
 		writeTo(destination);
 		canSave.setValue(destination.canWrite());
 		destination.subscribe(this);
+        this.schema = schema;
 	}
 	
+
 	@Override
 	protected String transform(SynopticDescription value) {
 		// Converts the raw synoptic description into XML
 		try {
-			return XMLUtil.toXml(value);
+            return XMLUtil.toXml(value, SynopticDescription.class, schema.getValue());
```

Comment:
I think that logging is the most appropriate here. If the schema is not available it doesn't necessarily mean that the GUI is saving synoptics badly and even if it is the Blockserver will guard against anything being too wrong. You're right that we, as support/devs, should be aware if this has happened though.